### ISSUE TITLE
Image Tint Issues

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#1919]
The image tint on the listing card image should be darkened to help with a11y as the text can be hard to read if it wraps onto two lines.

Example on Bloom:
![Screen Shot 2021-10-04 at 10.10.52 AM.png](https://images.zenhubusercontent.com/601c7ede5322cf8ac29d50c4/ed82a200-ea57-4248-91a8-db33700a84b3)
Example on DAHLIA:
![Screen Shot 2021-10-04 at 10.11.24 AM.png](https://images.zenhubusercontent.com/601c7ede5322cf8ac29d50c4/121592f2-fbfd-407d-b00e-2a5f27d9b957)

Additionally when the image is small, sometimes the tint doesn't reach the bottom of the image
![Screen Shot 2021-10-04 at 10.13.36 AM.png](https://images.zenhubusercontent.com/601c7ede5322cf8ac29d50c4/8fc4ee63-88d8-4d5f-9afd-671ee43201ca)
